Cargo.toml

```diff
@@ -74,3 +77,6 @@ autocfg = "0.1"
 
 [package.metadata.docs.rs]
 all-features = true
+
+[patch.crates-io]
+rand_core = { path = "rand_core", version = "0.3", default-features = false }
```

Comment:
When filing the upstream PR, make sure to mention in the PR description why this works